crossbuilder

```diff
@@ -648,6 +648,10 @@ if stat --file-system $HOME | grep ecrypt ; then
     ENCRYPTED_HOME=1
 fi
 
+if [ `which lxc` = "/snap/bin/lxc" ]; then
```

Comment:
Can you please explain (in the commit message or in a comment) why exactly this is needed? What happens otherwise?

Reply:
The Snap version of LXD requires privileged containers as newuidmap and newgidmap are not shipped, which would be required for unprivileged containers.